experimental/uefi/app/src/tests/remote_attestation.rs

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Integration Test of Remote Attestation in UEFI
+//! This tests that remote attestion works inside the UEFI app. While the test
+//! code is identical to (a subset of) the tests in the remote attestation crate
+//! they here utilize the qemu runner configured in the UEFI app. This means
+//! that test code actually compiled to a UEFI target, which changes the
+//! underlying implementation of the remote attestation crate.
+//! TODO(#2654): It would be preferable to remove the test here, and instead
+//! run the tests in the oak_remote_attestation crate itself for both standard
+//! and UEFI targets. Due to concerns related to the workspace this is presently
+//! not possible. Ref: https://github.com/project-oak/oak/issues/2654
```

Comment:
:( don't love repeating the test here, but imo the pragmatic choice for now

Reply:
I agree that it is a good pragmatic choice for now. Once we have an end-to-end example with a connection from a client that exercises this functionality the test could be removed.